packages/storefront/src/pages/must-know/definition-of-done.md

```diff
@@ -0,0 +1,33 @@
+# Definition Of Done
+
+We value high quality and transparency.  
+Thus we consider a task or ticket only done when all of the following criteria are fulfilled as our **Definition of
+Done** (DOD).
+
+<TableOfContents></TableOfContents>
+
+## Current Criteria
+
```

Comment:
Maybe we should also add things like, code is written with: 
- Semantic HTML
- Smallest possible DOM hierarchy
- Modern CSS techniques producing as less CSS as possible
- In general having everything properly typed, abstracted, tested and maintainable
…